crates/sqlbuiltins/Cargo.toml

```diff
@@ -33,3 +33,12 @@ memoize = { version = "0.4.2", features = ["full"] }
 async-openai = "0.18.2"
 tokio.workspace = true
 reqwest.workspace = true
+candle-nn = "0.3.3"
+candle-core = "0.3.3"
+candle-transformers = "0.3.3"
+hf-hub = "0.3.2"
+tokenizers = "0.15.1"
+serde_json.workspace = true
+lance-linalg =  { git = "https://github.com/lancedb/lance", rev = "310d79eccf93f3c6a48c162c575918cdba13faec" }
```

Comment:
does it make sense to (eventually) put the extension stuff in a different crate?

Reply:
Yeah i think our end goal should be to load most of them as dylibs and only have a select few _(if any)_ that are actually embedded into the binary but disabled.